src/main/java/se/sundsvall/ai/flow/api/FlowResource.java

```diff
@@ -52,17 +60,31 @@ class FlowResource {
 				useReturnTypeSchema = true)
 		})
 	@GetMapping
-	ResponseEntity<List<FlowInfo>> getAllFlows(
+	ResponseEntity<Flows> getFlows(
 		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId) {
-		var flows = flowRegistry.getAllFlows().stream()
-			.map(flow -> new FlowInfo(flow.getId(), flow.getName(), flow.getDescription(), flow.getDefaultTemplateId()))
-			.toList();
+		return ok(flowService.getFlows());
+	}
 
-		return ok(flows);
+	@Operation(summary = "Get a flow by name and version",
+		responses = {
+			@ApiResponse(
+				responseCode = "200",
+				description = "Ok",
+				useReturnTypeSchema = true),
+			@ApiResponse(
```

Comment:
missing mediatype

Reply:
(if you dont specify mediatype for 4xx and 5xx ApiResponses they will default to the default. which in this case is APPLICATION_JSON_VALUE, as declared on row 34)